examples/with_winit/src/lib.rs

```diff
@@ -356,6 +374,7 @@ fn run(
                     .unwrap_or(Color::BLACK),
                 width,
                 height,
+                antialiasing_method: aa_config,
```

Comment:
I'd probably inline this (so that the comment above about base_colour logic is in the right place)

Reply:
`aa_config` gets reused below so I wanted to have a single binding for it. I moved things around so that the comment is positioned better.